src/IRrecv.h

```diff
@@ -663,6 +663,12 @@ class IRrecv {
                            const uint16_t nbits = kMultibracketsBits,
                            const bool strict = true);
 #endif  // DECODE_MULTIBRACKETS
+#if DECODE_TECHNIBEL_AC
+  bool decodeTechnibelAc(decode_results *results,
+                     uint16_t offset = kStartOffset,
+                     const uint16_t nbits = kTechnibelAcBits,
+                     const bool strict = true);
```

Comment:
Style nitpick: Please align these lines to start under the `d` of decode_results. i.e. after the `(` column.